building.md

```diff
@@ -1,30 +1,22 @@
 ## Building the plugin
 
-We use gradle to build the plugin, and require version 2.1 or greater (`gradle -version`).
+We use ant to build the plugin; to check your ant install, run `ant -version`.
 
 To build the plugin, type:
 
 ```
-gradle build
+ant build
 ```
 
 Artifacts are written into the `build/` directory. For example:
 
 ```
-build/distributions/Flutter-0.0.1.zip
+build/flutter-intellij.jar
 ```
 
-## More information
-
-Gradle can be installed via `brew` on the Mac (`brew install gradle`), or via `SDKMAN` as
-described on the gradle [web site](http://www.gradle.org/downloads).
-
-Documentation for the options in `gradle.properties` is available at
-[github.com/JetBrains/gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin).
```

Comment:
Maybe we can add instructions for installing ant here?  Is `brew` the way to go?